python/samples/concepts/reasoning/simple_reasoning.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
```

Comment:
This is great, thanks for adding the sample. Can we please update the concept sample README with this new "reasoning" category? https://github.com/microsoft/semantic-kernel/blob/main/python/samples/concepts/README.md

Reply:
@moonbox3 
ok. Is it like this: https://github.com/microsoft/semantic-kernel/commit/fd3e0dc96a861331f73000009dc55d6cde5dac33#diff-f5e406ce9b1b9d01d54d2380c88086923087415cda950c90bbee37020c856a03R25